### PR TITLE
Prismexp appyter

### PR DESCRIPTION
**Pull request by lachmann12**
Added the PrismEXP appyter folder to the appyter cataloge

### CODE REVIEW

File: appyters/PrismEXP/PrismEXP.ipynb [hunk 1]
```diff
@@ -0,0 +1,522 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%appyter init\n",
+    "import os, sys; sys.path.insert(0, os.path.realpath('..'))\n",
```
Comment (u8sand) [inline appyters/PrismEXP/PrismEXP.ipynb:10]: FYI, this is only relevant for the example appyter because it runs in the appyter directory and is using the source code for appyter instead of the appyter module. If you have appyter installed with pip however this is irrelevant and just loading the parent directory into PYTHONPATH. It could be removed.

File: appyters/PrismEXP/PrismEXP.ipynb [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+    "    subtitle='Choose a target gene for gene function prediction.',\n",
+    "    name='GENE',\n",
+    "    img='enrichr-icon.png',\n",
+    ") %}\n",
+    "\n",
+    "{% set gene_symbol = StringField(name='gene_symbol', label='Gene Symbol', default='MAPK1', description='Enter an official gene symbol for functional prediction.', section = 'GENE') %}\n",
+    "\n",
+    "{% do SectionField(\n",
+    "    title='GMT Selection',\n",
+    "    subtitle='Uplpoad or select GMT file.',\n",
```
Comment (u8sand) [inline appyters/PrismEXP/PrismEXP.ipynb:34]: You misspelled upload here

File: appyters/PrismEXP/PrismEXP.ipynb [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter markdown\n",
+    "\n",
+    "<h1> <img align=\"left\" width=200 src=\"https://mssm-prismx.s3.amazonaws.com/images/prismxsmall.png\">\n",
```
Comment (u8sand) [inline appyters/PrismEXP/PrismEXP.ipynb:79]: You might want to add `class="img-fluid"` to play well with bootstrap, typically I do this automagically when I render markdown.

File: appyters/PrismEXP/static/AutocompleteField.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
+// Adapted from W3schools
+
+function runall($, name, file_path) {
```
Comment (u8sand) [inline appyters/PrismEXP/static/AutocompleteField.js:3]: If you're not using this field, you can remove this file artifact.

File: appyters/PrismEXP/templates/fields/textfield.j2 [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<div class="row px-4 px-lg-3 pb-4">
```
Comment (u8sand) [inline appyters/PrismEXP/templates/fields/textfield.j2:1]: Is this overriding `TextField`? If so, please make sure it is the right case. Some operating systems care more about case sensitivity than others..

File: appyters/PrismEXP/templates/form.j2 [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+        var target = $(this).attr('target')
+        var value = self.find('[role="tab"].active').attr('target')
+        $(target).val(value);
+      })
+      // Submit form
+      this.submit()
+    })
+  })
+})
+</script>
```
Comment (u8sand) [inline appyters/PrismEXP/templates/form.j2:119]: This script block seems irrelevant given that all of these things are defined in `{{ super() }}`  It was necessary in Drugmonizome_ML Because of tab-field but that is now a part of appyter.

File: appyters/PrismEXP/templates/fields/textfield.j2 (hunk 5 above)
Comment (lachmann12) [reply appyters/PrismEXP/templates/fields/textfield.j2:1]: removed, it's not used right now

File: appyters/PrismEXP/static/AutocompleteField.js (hunk 4 above)
Comment (lachmann12) [reply appyters/PrismEXP/static/AutocompleteField.js:3]: also not used

File: appyters/PrismEXP/PrismEXP.ipynb (hunk 3 above)
Comment (lachmann12) [reply appyters/PrismEXP/PrismEXP.ipynb:79]: this part behaved very strangely and would result in completely wrong formatting just by adding white spaces at the wrong place.

File: appyters/PrismEXP/README.md [hunk 7]
```diff
@@ -1,6 +1,9 @@
 # PrismEXP
+## Gene Function Prediction
 
-![PrismEXP](static/featured-image.png?raw=true "PrismEXP")
+<p align="center">
+<img src="static/featured-image.png" width=800 />
```
Comment (u8sand) [inline appyters/PrismEXP/README.md:5]: Please use `class="img-fluid"` and don't explicitly set the width, or make it `100%`, this is relevant for how it will get displayed in the catalog landing page

File: appyters/PrismEXP/appyter.json [hunk 8]
```diff
@@ -14,7 +14,9 @@
     "url": "https://github.com/MaayanLab/appyter-catalog",
     "tags": [
       "PrismEXP",
-      "gene function prediction"
+      "gene function prediction",
```
Comment (u8sand) [inline appyters/PrismEXP/appyter.json:17]: Consider adopting tags that already exist on appyters https://appyters.maayanlab.cloud

File: appyters/PrismEXP/appyter.json [hunk 9]
```diff
@@ -14,7 +14,9 @@
     "url": "https://github.com/MaayanLab/appyter-catalog",
```
Comment (u8sand) [inline appyters/PrismEXP/appyter.json:14]: Does PrismEXP have its own url / repo?
